[jit] fix missing lock in profiling graph compilation

zdevito opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#29886 [jit] fix missing lock in profiling graph compilation**

Fixes https://github.com/pytorch/pytorch/issues/29764

Differential Revision: [D18523903](https://our.internmc.facebook.com/intern/diff/D18523903)

zdevito committed:
[jit] fix missing lock in profiling graph compilation

Fixes https://github.com/pytorch/pytorch/issues/29764

kostmo commented:
## CircleCI build failures summary
As of commit 4072672:
* **1/1** failures introduced in this PR
* **0/1** recognized as flaky



Here are the [reasons each build failed](https://dr.pytorch.org/commit-details.html?sha1=407267294f4b4118db12f0033cb72660c461ecf2):

| Job | Step | Log excerpt |
| --- | --- | --- |


---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

bobforgithub commented:
unsubscribe

在 2019-11-15 11:08:44，"Karl Ostmo" <notifications@github.com> 写道：

CircleCI build failures summary

As of commit 4072672:

1/1 failures introduced in this PR
0/1 recognized as flaky

Here are the reasons each build failed:

| Job | Step | Log excerpt |

This comment was automatically generated by Dr. CI.
Follow this link to opt-out of these comments for your Pull Requests.

Please report bugs/suggestions on the GitHub issue tracker.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.

zdevito committed:
Update on "[jit] fix missing lock in profiling graph compilation"

[jit] fix missing lock in profiling graph compilation

Fixes https://github.com/pytorch/pytorch/issues/29764

gh-metadata: pytorch pytorch 29886 gh/zdevito/134/head

zdevito committed:
Update on "[jit] fix missing lock in profiling graph compilation"

[jit] fix missing lock in profiling graph compilation

Fixes https://github.com/pytorch/pytorch/issues/29764

gh-metadata: pytorch pytorch 29886 gh/zdevito/134/head
